camel/utils/commons.py

```diff
@@ -241,3 +182,81 @@ def check_server_running(server_url: str) -> bool:
 
     # if the port is open, the result should be 0.
     return result == 0
+
+
+def get_openai_function_schema(func: Callable) -> Dict[str, any]:
```

Comment:
I added type annotations here so some new mypy errors detected. Could you please fix it? Also Please add docstings for `get_openai_function_schema` and `get_openai_tool_schema`.

Reply:
Okay, I will fix it and add docstrings